src/discord.coffee

```diff
@@ -1,91 +1,187 @@
+# Description:
+#   Adapter for Hubot to communicate on Discord
+#
+# Commands:
+#   None
+#
+# Configuration:
+#   HUBOT_DISCORD_TOKEN          - authentication token for bot
+#   HUBOT_DISCORD_STATUS_MSG     - Status message to set for "currently playing game"
+#   HUBOT_DISCORD_CARBON_TOKEN   - Carbonitex.net bot authentication
+#   HUBOT_DISCORD_BOTS_WEB_USER  - bots.discord.pw user id for bot
+#   HUBOT_DISCORD_BOTS_WEB_TOKEN - bots.discord.pw auth token
+#   
+# Notes:
+# 
 try
     {Robot, Adapter, EnterMessage, LeaveMessage, TopicMessage, TextMessage}  = require 'hubot'
 catch
     prequire = require( 'parent-require' )
     {Robot, Adapter, EnterMessage, LeaveMessage, TopicMessage, TextMessage}  = prequire 'hubot'
-Discord = require "discord.js"
 
-rooms = {}
+Discord             = require "discord.js"
```

Comment:
The space padding on this line and below are to line up the `=` to make things a bit easier to read.